build/Versions.props

```diff
@@ -1,10 +1,10 @@
 <Project>
   <PropertyGroup>
-    <VersionBase>2.3.0</VersionBase>
-    <PreReleaseVersionLabel>beta2</PreReleaseVersionLabel>
+    <VersionBase>2.6.0</VersionBase>
```

Comment:
@tannergooding @jasonmalinowski by 2.5, you mean roslyn version? can you give us specific version you guys will move onto?